src/platform/update_modules/zephyr/image/update-module.c

```diff
@@ -298,6 +302,14 @@ mender_zephyr_image_abort_deployment(MENDER_NDEBUG_UNUSED mender_update_state_t
         return ret;
     }
     artifact_had_payload = false;
+
+    int rc;
+    if (boot_is_img_confirmed()) {
+        if ((rc = boot_erase_img_bank(MENDER_SLOT_PARTITION_1)) < 0) {
+            mender_log_error("Failed to erase second slot %d", rc);
```

Comment:
Can we get some explanatory message for the error code? Otherwise, let's at least put it in some parentheses/brackets.